hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -586,9 +610,23 @@ extern "C" unsigned otbn_model_step(OtbnModel *model, svLogic start_i,
   if (!(status & RUNNING_BIT))
     return status;
 
+  bool edn_rnd_data_valid = false;
+  bool edn_urnd_data_valid = false;
+  uint32_t edn_rnd_data[8];
+
+  if (edn_rnd_data_valid_i) {
+    edn_rnd_data_valid = true;
+    set_rnd_data(edn_rnd_data, edn_rnd_data_i);
+  }
+
+  if (edn_urnd_data_valid_i) {
+    edn_urnd_data_valid = true;
+  }
```

Comment:
This feels redundant, just inline the _i variable?

Reply:
Yeah probably better, as it's a svLogic -> boolean conversion I did it this way to make it a bit more explicit (plus it matches how we're using `edn_rnd_data_valid_i`) but the `if` is still using an implicit `bool` conversion anyway.

Reply:
As with the other note, maybe just pass this through. Also, could we add an X check here?